.gitignore

```diff
@@ -14,3 +14,4 @@
 /firebase_functions/appengine
 
 *.log
+yarn.lock
```

Comment:
please revert this change, this file should be update every time a package is changed or added